### PR TITLE
keytool: add page

### PR DESCRIPTION
**Pull request by martincalvert**
- **keytool shipped with Java 1.8**

### CODE REVIEW

File: pages/common/keytool.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# keytool
+
+> Keytool is a certificate management utility included with Java.
+> More information: <https://docs.oracle.com/javase/8/docs/technotes/tools/windows/keytool.html>.
```
Comment (kbdharun) [inline pages/common/keytool.md:4]:
```suggestion
> More information: <https://docs.oracle.com/en/java/javase/19/docs/specs/man/keytool.html>.
```
I would suggest updating to a more recent version of documentation.

File: pages/common/keytool.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Create a keystore:
+
+`keytool -genkey -v -keystore {{keystore_name}}.keystore -alias {{key_name}}`
```
Comment (kbdharun) [inline pages/common/keytool.md:8]:
```suggestion
`keytool -genkeypair -v -keystore {{path/to/file}}.keystore -alias {{key_name}}`
```

Ref docs:
> This command was named -genkey in earlier releases. The old name is still supported in this release. The new name, -genkeypair, is preferred going forward.

Assuming Keystore is a file I would suggest this.

File: pages/common/keytool.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Change a keystore password:
+
+`keytool -storepasswd -keystore {{keystore_name}}.keystore`
```
Comment (kbdharun) [inline pages/common/keytool.md:12]:
```suggestion
`keytool -storepasswd -keystore {{path/to/file}}.keystore`
```

File: pages/common/keytool.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Change a key's password inside a keystore:
+
+`keytool -keypasswd  -alias {{key_name}} -keystore {{keystore_name}}.keystore`
```
Comment (kbdharun) [inline pages/common/keytool.md:16]:
```suggestion
`keytool -keypasswd  -alias {{key_name}} -keystore {{path/to/file}}.keystore`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by martincalvert** [comment]
Updated with PR comments. If approved ill squash rebase the 2 commits into `key tool: add page`

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Comment by kbdharun** [comment]
> Updated with PR comments. If approved ill squash rebase the 2 commits into `key tool: add page`

We squash and merge it by default so it isn't required.

File: pages/common/keytool.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# keytool
+
+> Keytool is a certificate management utility included with Java.
+> More information: <https://docs.oracle.com/en/java/javase/19/docs/specs/man/keytool.html>.
+
+- Create a keystore:
+
+`keytool -genkeypair -v -keystore {{path/to/file}}.keystore -alias {{key_name}}`
```
Comment (EmilySeville7cfg) [inline pages/common/keytool.md:8]:
```suggestion
- Create a keystore:

`keytool -genkeypair -v -keystore {{path/to/file.keystore}} -alias {{key_name}}`
```

Optional fix: placeholder can be expanded.

File: pages/common/keytool.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+- Change a keystore password:
+
+`keytool -storepasswd -keystore {{path/to/file}}.keystore`
```
Comment (EmilySeville7cfg) [inline pages/common/keytool.md:12]:
```suggestion
- Change a keystore password:

`keytool -storepasswd -keystore {{path/to/file.keystore}}`
```

The same issue as above.

File: pages/common/keytool.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+
+- Change a key's password inside a keystore:
+
+`keytool -keypasswd  -alias {{key_name}} -keystore {{path/to/file}}.keystore`
```
Comment (EmilySeville7cfg) [inline pages/common/keytool.md:16]:
```suggestion
- Change a key's password inside a specific keystore:

`keytool -keypasswd  -alias {{key_name}} -keystore {{path/to/file.keystore}}`
```

Almost the same issue as above.
Optional fix: `specific` may sound better.

**Review by EmilySeville7cfg — APPROVED** [review]
No mandatory fixes found, approved.
Thanks for a contribution! ❤️

**Review by blueskyson — APPROVED** [review]

**Pull request merged**